diff --git a/OB-Xd Linux.jucer b/OB-Xd Linux.jucer
index 8f84a97..3c365bb 100644
--- a/OB-Xd Linux.jucer	
+++ b/OB-Xd Linux.jucer	
@@ -9,8 +9,8 @@
               pluginAUExportPrefix="" pluginRTASCategory="2048" aaxIdentifier="com.discodsp.obxd"
               companyName="discoDSP" companyWebsite="https://www.discodsp.com/"
               pluginIsMidiEffectPlugin="0" pluginCharacteristicsValue="pluginIsSynth,pluginWantsMidiIn"
-              pluginFormats="buildAU,buildLV2,buildStandalone,buildVST,buildVST3"
-              buildVST="1" buildVST3="1" buildAU="1" buildAUv3="0" buildRTAS="0"
+              pluginFormats="buildAU,buildLV2,buildStandalone,buildVST3"
+              buildVST="0" buildVST3="1" buildAU="1" buildAUv3="0" buildRTAS="0"
               buildAAX="0" buildStandalone="1" enableIAA="0" jucerFormatVersion="1"
               companyCopyright="discoDSP" companyEmail="contactus@discodsp.com"
               pluginAAXCategory="2048" pluginVSTCategory="kPlugCategSynth"
@@ -91,27 +91,27 @@
     </GROUP>
   </MAINGROUP>
   <EXPORTFORMATS>
-    <LINUX_MAKE targetFolder="Builds/LinuxMakefile" extraLinkerFlags="-no-pie -lcurl"
-                vstLegacyFolder="Modules/vstsdk2.4" extraDefs="JUCE_MODAL_LOOPS_PERMITTED=1">
+    <LINUX_MAKE targetFolder="Builds/LinuxMakefile" extraLinkerFlags="-no-pie"
+                vstLegacyFolder="Modules/vstsdk2.4" vst3Folder="Modules/vstsdk3" extraDefs="JUCE_MODAL_LOOPS_PERMITTED=1">
       <CONFIGURATIONS>
         <CONFIGURATION name="Release64" libraryPath="/usr/X11R6/lib/" isDebug="0" optimisation="3"
-                       targetName="OB-Xd" linuxArchitecture="-m64" headerPath="../../JuceLibraryCode&#10;../../Source&#10;../Modules/vstsdk2.4&#10;/usr/include/freetype2&#10;/usr/include"
+                       targetName="OB-Xd" linuxArchitecture="" headerPath="../../JuceLibraryCode&#10;../../Source&#10;/usr/include/freetype2&#10;/usr/include"
                        linkTimeOptimisation="1" enablePluginBinaryCopyStep="0"/>
       </CONFIGURATIONS>
       <MODULEPATHS>
-        <MODULEPATH id="juce_opengl" path="Modules"/>
-        <MODULEPATH id="juce_gui_extra" path="Modules"/>
-        <MODULEPATH id="juce_gui_basics" path="Modules"/>
-        <MODULEPATH id="juce_graphics" path="Modules"/>
-        <MODULEPATH id="juce_events" path="Modules"/>
-        <MODULEPATH id="juce_data_structures" path="Modules"/>
-        <MODULEPATH id="juce_core" path="Modules"/>
-        <MODULEPATH id="juce_audio_utils" path="Modules"/>
-        <MODULEPATH id="juce_audio_processors" path="Modules"/>
-        <MODULEPATH id="juce_audio_plugin_client" path="Modules"/>
-        <MODULEPATH id="juce_audio_formats" path="Modules"/>
-        <MODULEPATH id="juce_audio_devices" path="Modules"/>
-        <MODULEPATH id="juce_audio_basics" path="Modules"/>
+        <MODULEPATH id="juce_opengl" path="JUCE/modules"/>
+        <MODULEPATH id="juce_gui_extra" path="JUCE/modules"/>
+        <MODULEPATH id="juce_gui_basics" path="JUCE/modules"/>
+        <MODULEPATH id="juce_graphics" path="JUCE/modules"/>
+        <MODULEPATH id="juce_events" path="JUCE/modules"/>
+        <MODULEPATH id="juce_data_structures" path="JUCE/modules"/>
+        <MODULEPATH id="juce_core" path="JUCE/modules"/>
+        <MODULEPATH id="juce_audio_utils" path="JUCE/modules"/>
+        <MODULEPATH id="juce_audio_processors" path="JUCE/modules"/>
+        <MODULEPATH id="juce_audio_plugin_client" path="JUCE/modules"/>
+        <MODULEPATH id="juce_audio_formats" path="JUCE/modules"/>
+        <MODULEPATH id="juce_audio_devices" path="JUCE/modules"/>
+        <MODULEPATH id="juce_audio_basics" path="JUCE/modules"/>
       </MODULEPATHS>
     </LINUX_MAKE>
   </EXPORTFORMATS>
@@ -132,6 +132,7 @@
     <MODULES id="juce_opengl" showAllCode="1" useLocalCopy="0" useGlobalPath="0"/>
   </MODULES>
   <JUCEOPTIONS JUCE_QUICKTIME="disabled" JUCE_WEB_BROWSER="0" JUCE_ASIO="1"
+               JUCE_VST3_CAN_REPLACE_VST2="0"
                JUCE_JACK="1" JUCE_USE_CURL="0"/>
   <LIVE_SETTINGS>
     <OSX/>
